PWGHF/TableProducer/candidateSelectorLc.cxx

```diff
@@ -201,6 +201,26 @@ struct HfCandidateSelectorLc {
       return false;
     }
 
+    // candidate decay length XY
+    if (candidate.decayLengthXY() <= cuts->get(pTBin, "decLengthXY")) {
+      return false;
+    }
+
+    // candidate normalized decay length XY
+    if (candidate.decayLengthXYNormalised() < cuts->get(pTBin, "normDecLXY")) {
+      return false;
+    }
+
+    // candidate ct
+    if (hfHelper.ctXic(candidate) > cuts->get(pTBin, "ct")) {
+      return false;
+    }
```

Comment:
do you really need a cut on `hfHelper.ctXic(candidate)`? If yes, I'd anyway remove it from the default selections for Lambdac, and eventually add a specific `Configurable` in the selector. I do not like the idea to have a Xic-related cut in the code of Lambdac

Reply:
@mfaggin Good catch!